README.md

```diff
@@ -504,30 +512,60 @@ You can fetch data from Storyblok directly using queries. To add the query:
 
-## GitHub-based blog workflow (Labs blog) 💻
+## GitHub-based blog workflow (Labs and Consulting blogs) 💻
 
-All the **Quansight Labs** blog posts are located inside `apps/labs/post`,
-and therefore, any new posts _must_ be added to this same folder.
+All the blog posts for **both websites** are located inside `./apps/labs/posts`
+and `./apps/consulting/posts`, and therefore, any new posts _must_ be added to
+these same folders.
 
 > **Note**
 > For now, all posts should be
 > contributed using a branch-and-merge strategy within the website repo itself,
 > instead of a fork-and-merge strategy. This may change in the future.
 
-Every post is a `.md` or [`.mdx` file](https://mdxjs.com/docs/using-mdx/). The
-`posts` directory also contains a [`categories.json`
-file](./apps/labs/posts/categories.json) containing the posts categories.
-
-The `categories.json` file is also used for displaying category filters on the
-`/blog` page so after adding a new category, it will also be visible on that
-page.
-
-For more details about `.mdx` please see:
+Every post is a `.md` or [`.mdx` file](https://mdxjs.com/docs/using-mdx/). For
+more details about `.mdx` please see:
 
 - <https://mdxjs.com/>
 - <https://github.com/hashicorp/next-mdx-remote>
 
-### Structure of the blog post
+Each `posts` directory also contains a `categories.json` file
+([Labs file](./apps/labs/posts/categories.json) and
+[Consulting file](./apps/consulting/posts/categories.json)) containing the post
+categories. The formats of these `categories.json` files are currently **not**
+the same:
+
+- For the Labs blog, `categories.json` is an array of case-sensitive strings.
+  These strings are used:
+  - As-is in the rendered category list on the main `/blog` page
+  - In a custom slugified format (spaces replaced with `+`) when filtering
+    `/blog` by category
+    - For example, the category `"Beyond PyData"` is slugified as
+      `"Beyond+PyData`"
+    - This category filter is currently **case-sensitive**—URLs with mismatched
+      case in the `category` query parameter will return **no results**
+- For the Consulting blog, `categories.json` is a list of objects, where each
+  object has two `string` properties:
+  - `name`
+    - Should be written with natural capitalization and whitespace
+    - Used as-is (case and whitespace sensitive, and _without_ surrounding quotes) in the post `category` metadata field to associate a
+      post with one or more categories
+    - Converted to all caps for use in the category-selection dropdown on the
+      `/library` page
+  - `value`
+    - Should be written in slugified form and in all lowercase
+      - NOTE: Currently one category, `Infrastructure & HPC`, has a capitalized
+        `value` due to an oversight. We plan to fix this.
+    - Used as-is (**case sensitive**) in the `category` query parameter to
+      `/library`—URLs with mismatched case in the `category` query parameter
+      will return **no results**
```

Comment:
Is this level of detail appropriate here. I know, I know, this whole README is overly detailed. But honestly, this explanation of the `categories.json` files makes my head spin. I hate the fact that JSON doesn't support comments.

I think this is valuable information. I'm just wondering if we could expose it somewhere more useful. Maybe not. Just wondering.

I think what this section is really screaming is that we need to simplify/fix the system. So maybe it would make sense to move all of this detail into an issue and link to the issue from the README?